gcloud_bigtable/row.py

```diff
@@ -416,12 +419,13 @@ def _commit_mutate(self, timeout_seconds=None):
             mutations=mutations_list,
         )
         timeout_seconds = timeout_seconds or self.timeout_seconds
-        response = self.client.data_stub.MutateRow.async(request_pb,
-                                                         timeout_seconds)
         # We expect a `._generated.empty_pb2.Empty`.
-        response.result()
+        if async:
+            self.client.data_stub.MutateRow.async(request_pb, timeout_seconds).result()
```

Comment:
Good to know.

I'll probably just ditch `.async` altogether once the library is upgraded to the `beta` version of gRPC core. (Maybe it already works just fine with `beta`, since you mentioned `0.11` works.)

----

@nathanielmanistaatgoogle I'd love to discuss reasons to use `.async` in a library at a later time.